### ISSUE TITLE
Upgrade from C110 -> C111 on macOS x64 via 1.50.x - Nightly (Checking C111)

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#28684]
As per process, QA runs through the following cases to ensure that the major chromium bump that's about to get merged into `master` doesn't regress the `Nightly` channel. Once the major chromoium bump is merged into `master`, QA will run through a full manual pass on `Windows` & `Android`.

### Startup & Components

- [x] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler`, `Wireshark` or `LittleSnitch` (or a similar application)
  - [x] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
- [x] Remove the following component folders and ensure that they're being re-downloaded after restarting the browser:
  - [x] `afalakplffnnnlkncjhbmahjfjhmlkal`: `AutoplayWhitelist.dat`, `ExtensionWhitelist.dat`, `ReferrerWhitelist.json` and `Greaselion.json`
  - [x] `CertificateRevocation`
  - [x] `gccbbckogglekeggclmmekihdgdpdgoe`: (Sponsored New Tab Images)
  - [x] `gkboaolpopklhgplhaaiboijnklogmbc`: Brave Ad Block List Catalog
  - [x] `iodkpdagapdfkphljnddpjlldadblomo`: Brave Ad Block Updater
  - [x] `mfddibmblmbccpadfndgakiopmmhebop`: Brave Ad Block List Catalog
  - [x] `oofiananboodjbbmdelgdommihjbkfag`: HTTPSE
  - [x] `Safe Browsing`
- [x] Restart the browser, load `brave://components`, wait for 8 mins and verify that no component shows any errors

**Note:** Always double check `brave://components` to make sure there's no errors/missing version numbers

### Upgrade

- [x] Make sure that data from the last version appears in the new version OK
- [x] Ensure that `brave://version` lists the expected Brave & Chromium versions
- [x] With data from the last version, verify that:
  - [x] Bookmarks on the bookmark toolbar and bookmark folders can be opened
  - [x] Cookies are preserved
  - [x] Installed extensions are retained and work correctly
  - [x] Opened tabs can be reloaded
  - [x] Stored passwords are preserved
  - [x] Sync chain created in previous version is retained
  - [x] Social media-blocking buttons changes are retained
  - [x] Custom filters under brave://settings/shields/filters are retained
  - [x] Custom lists under brave://settings/shields/filters are retained
  - [x] Rewards
    - [x] BAT balance is retained
    - [x] Auto-contribute list is retained
    - [x] Both Tips and Monthly Contributions are retained
    - [x] Panel transactions list is retained
    - [x] Changes to rewards settings are retained
    - [x] Ensure that Auto Contribute is not being enabled when upgrading to a new version if AC was disabled
  - [x] Ads
    - [x] Both `Estimated pending rewards` & `Ad notifications received this month` are retained
    - [x] Changes to ads settings are retained
    - [x] Ensure that ads are not being enabled when upgrading to a new version if they were disabled
    - [x] Ensure that ads are not disabled when upgrading to a new version if they were enabled

**Comment by stephendonner** [comment]
## Verified `PASSED` using 

Brave | 1.50.56 Chromium: 110.0.5481.104 (Official Build) nightly (x86_64)
-- | --
Revision | 46de4a7f41979e829b430bc1ee30ef483aa227ac-refs/branch-heads/5481_77@{#19}
OS | macOS Version 11.7.4 (Build 20G1120)

**Comment by stephendonner** [comment]
Screwed up and tested this on `CR110` rather than:

1.50.55 Chromium: 111.0.5563.33 (Official Build) nightly (x86_64)

Re-testing now, sorry for the noise.

**Comment by stephendonner** [comment]
## Verified `PASSED` (for real this time 👍 ) using

Brave | 1.50.55 Chromium: 111.0.5563.33 (Official Build) nightly (x86_64)
-- | --
Revision | 19bd6d0186b7912604e18191826dab9d1df00e2e-refs/branch-heads/5563@{#557}
OS | macOS Version 11.7.4 (Build 20G1120)

<img width="1489" alt="Screen Shot 2023-02-22 at 1 26 26 PM" src="https://user-images.githubusercontent.com/387249/220763179-8fa804a9-3cc2-47f7-a505-04d945fd3ca2.png">
